arci-ros/src/msg.rs

```diff
@@ -16,6 +16,7 @@ ros_nalgebra::rosmsg_include!(
     sensor_msgs / JointState,
     sensor_msgs / Joy,
     std_msgs / Header,
+    std_msgs / Float64,
```

Comment:
Please sort alphabetically.

```suggestion
    std_msgs / Float64,
    std_msgs / Header,
```